### PR TITLE
add VaultFactory, Tokenlist, lbp testfile

### PR DESCRIPTION
**Pull request by Zitzak**
##  What's new
- I created a VaultFactory class to easily create the Vault
- I added a TokenList class to create multiple tokens at once. This class can also be called with an integer of how many tokens you want.
- I added a lbpFactory method to the test-init.js file

### Issues
- I have trouble getting the artifact imported to create the Authorizer in file VaultFactory -> line 18
- I don't get line 30 to work in file TokenList

Also, I commented out the other test files for convenience, so I don't have to scroll as much everytime

### CODE REVIEW

**Comment by Zitzak** [comment]
Because of cleaning up the lbp.js, it didn't work properly. Fixed this

File: hardhat.config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require("@nomiclabs/hardhat-waffle");
 require("dotenv").config({path: "./.env"});
-require("hardhat-deploy");
+// require("hardhat-deploy");
```
Comment (kinspotting) [inline hardhat.config.js:3]: any reason why we keep it commented out? pls delete it or uncomment

File: test/test-init.js [hunk 2]
```diff
@@ -68,6 +71,21 @@ const seedMasterCopy = async (setup) => {
...
 
+const Lbp = (setup) => new ethers.ContractFactory(LBP.abi, LBP.bytecode, setup.roles.root);
+
+const deployLBPFactory = async (setup) => {
+    const LBPFactory_Factory = new ethers.ContractFactory(LBPFactory.abi, LBPFactory.bytecode, setup.roles.root);
+    const lbpFactory = await LBPFactory_Factory.deploy(setup.vault.address);
+    return lbpFactory;
+}
+
+const deployVault = async (setup) => {
```
Comment (kinspotting) [inline test/test-init.js:82]: let's move Vault and LBPFactory to a single class, we can call it balancer f.e.

File: test/test-init.js [hunk 3]
```diff
@@ -91,6 +110,9 @@ module.exports = {
     gnosisProxy,
     seedFactory,
     seedMasterCopy,
+    deployLBPFactory,
```
Comment (kinspotting) [inline test/test-init.js:113]: also the naming decision here is not the best, should be something like `LBPFactory` in this case

File: test/integration/lbp.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+	let poolTokens;
+	let lbp;
+
+	context("test factory", async () => {
+		before('setup', async () => {
+			setup = await deploy();
+		});
+		it('deploys new LBP pool', async () => {
+			const receipt = await (await setup.lbpFactory.create(
+				"Test",
```
Comment (kinspotting) [inline test/integration/lbp.js:35]: pls don't use magic values and store all the parameters in a separate variable

**Review by kinspotting — CHANGES_REQUESTED** [review]
needs changes

File: hardhat.config.js (hunk 1 above)
Comment (Zitzak) [reply hardhat.config.js:3]: I guess I should also delete the `// require("hardhat-gas-reporter");` line that is commented below as well?

**Comment by Zitzak** [comment]
I also moved `test-init.js` to the helpers directory

File: hardhat.config.js (hunk 1 above)
Comment (Zitzak) [reply hardhat.config.js]: Done

File: test/test-init.js (hunk 2 above)
Comment (Zitzak) [reply test/test-init.js]: I also moved the LBP creating to the same file

File: test/test-init.js (hunk 3 above)
Comment (Zitzak) [reply test/test-init.js]: Done

File: test/integration/lbp.js (hunk 4 above)
Comment (Zitzak) [reply test/integration/lbp.js]: Changed all the magic values

File: test/helpers/balancer.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    const vault = await Vault_factory.deploy(constants.ZERO_ADDRESS, constants.ZERO_ADDRESS, 0, 0);
+
+    return vault;
+}
+
+const Lbp = (setup) => new ethers.ContractFactory(LBPArtifact.abi, LBPArtifact.bytecode, setup.roles.root);
+
+module.exports = {
+	LBPFactory,
+	deployVault,
```
Comment (kinspotting) [inline test/helpers/balancer.js:23]: pls rename to `Vault`

**Review by kinspotting — CHANGES_REQUESTED** [review]

File: test/helpers/balancer.js (hunk 5 above)
Comment (Zitzak) [reply test/helpers/balancer.js]: Changed the name

**Comment by Zitzak** [comment]
I added a `token.js` file that generates a list of deployed tokens to help with further pool testing.

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**